lib/apiservers/engine/backends/backends.go

```diff
@@ -109,6 +110,8 @@ func Init(portLayerAddr, product string, config *config.VirtualContainerHostConf
 	}
 
 	t := rc.New(portLayerAddr, "/", []string{"http"})
+	t.Consumers["application/x-tar"] = archive.TarConsumer()
```

Comment:
But the port layer does not have an endpoint that either consumes or produces that mime-type, so it looks its now being used.